tin/templates/base.html

```diff
@@ -78,7 +78,7 @@
       {% endif %}
       {% if docs_app %}
```

Comment:
`docs_app` doesn't "exist" anymore, right? You should just be able to get rid of this section (since docs now redirects outside of Tin).

Reply:
Ah missed that, nice catch :)